depthai_examples/ros2_src/stereo_inertial_publisher.cpp

```diff
@@ -143,205 +167,288 @@ int main(int argc, char** argv){
+    node->declare_parameter("exposure_height_stereo", cameraControl.stereo.region.at(3));
+    node->declare_parameter("exposure_compensation_stereo", cameraControl.stereo.compensation);
+    node->declare_parameter("exposure_time_us_stereo", cameraControl.stereo.time_us);
+    node->declare_parameter("exposure_iso_stereo", cameraControl.stereo.sensitivity_iso);
+
+    node->declare_parameter("focus_mode", cameraControl.focus_mode);
+    node->declare_parameter("focus_region_x", cameraControl.focus_region.at(0));
+    node->declare_parameter("focus_region_y", cameraControl.focus_region.at(1));
+    node->declare_parameter("focus_region_width", cameraControl.focus_region.at(2));
+    node->declare_parameter("focus_region_height", cameraControl.focus_region.at(3));
+
+    node->get_parameter("tf_prefix", tfPrefix);
+    node->get_parameter("mode", mode);
+    node->get_parameter("lrcheck", lrcheck);
+    node->get_parameter("extended", extended);
+    node->get_parameter("subpixel", subpixel);
+    node->get_parameter("rectify", rectify);
     node->get_parameter("depth_aligned", depth_aligned);
-    node->get_parameter("stereo_fps",    stereo_fps);
-    node->get_parameter("confidence",    confidence);
-    node->get_parameter("LRchecktresh",  LRchecktresh);
+    node->get_parameter("stereo_fps", stereo_fps);
+    node->get_parameter("confidence", confidence);
+    node->get_parameter("LRchecktresh", LRchecktresh);
     node->get_parameter("monoResolution", monoResolution);
 
-    if(mode == "depth"){
+    node->get_parameter("median_enable", postProcessing.median_enable);
+    node->get_parameter("median_mode", postProcessing.median_mode);
+    node->get_parameter("speckle_enable", postProcessing.speckle_enable);
+    node->get_parameter("speckle_range", postProcessing.speckle_range);
+    node->get_parameter("temporal_enable", postProcessing.temporal_enable);
+    node->get_parameter("temporal_mode", postProcessing.temporal_mode);
+    node->get_parameter("temporal_alpha", postProcessing.temporal_alpha);
+    node->get_parameter("temporal_delta", postProcessing.temporal_delta);
+    node->get_parameter("spatial_enable", postProcessing.spatial_enable);
+    node->get_parameter("spatial_radius", postProcessing.spatial_radius);
+    node->get_parameter("spatial_alpha", postProcessing.spatial_alpha);
+    node->get_parameter("spatial_delta", postProcessing.spatial_delta);
+    node->get_parameter("spatial_iterations", postProcessing.spatial_iterations);
+    node->get_parameter("threshold_enable", postProcessing.threshold_enable);
+    node->get_parameter("threshold_max", postProcessing.threshold_max);
+    node->get_parameter("threshold_min", postProcessing.threshold_min);
+    node->get_parameter("decimation_enable", postProcessing.decimation_enable);
+    node->get_parameter("decimation_mode", postProcessing.decimation_mode);
+    node->get_parameter("decimation_factor", postProcessing.decimation_factor);
```

Comment:
Same feeling. Do we ened all of these to be as public variables and set like this ?